app/views/admin/_orders_list.html.slim

```diff
@@ -5,23 +5,32 @@
     thead
       tr
         th Email
-        th His name
-        th Her name
-        th His story
-        th Her story
+        th Customer
+        th Created At
         th Status
+        th Domain Name
         th Action
     tbody
       - @service.each do |order|
         tr
           td = order.email
-          td = order.his_name
-          td = order.her_name
-          td = order.his_story
-          td = order.her_story
-          td = order.status
+          td = order.your_name
+          td = order.created_at.strftime("%d/%m/%Y - %H:%M:%S %p")
           td
-            .row
-              = link_to 'Edit', edit_admin_service_order_path(order) 
-            .row
+            - if order.status == "approve"
```

Comment:
1. using order.approve? or order.pending?... instead
2. It's better to move whole ``if block`` to helper instead. Remember to add rspec for helper :octocat: